### PR TITLE
DEV: makes <Chat::Admin::ExportMessage /> use glimmer

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import Component from "@ember/component";
+import Component from "@glimmer/component";
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
 import { popupAjaxError } from "discourse/lib/ajax-error";
 import I18n from "I18n";
 
-export default class ExportMessages extends Component {
+export default class ChatAdminExportMessages extends Component {
```
Comment (jjaffeux) [inline plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js:7]: That's a minor nitpick, but If possible I prefer to follow the namespace for the class name:

- component Chat::Admin::ExportMessages
- class ChatAdminExportMessages

File: plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.hbs [hunk 2]
```diff
@@ -5,6 +5,6 @@
     @label="chat.admin.export_messages.create_export"
     @title="chat.admin.export_messages.create_export"
     @class="btn-primary"
-    @action={{action this.exportMessages}}
+    @action={{this.exportMessages}}
```
Comment (jjaffeux) [inline plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.hbs:8]: action is not needed here

File: plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js (hunk 1 above)
Comment (AndrewPrigorshnev) [reply plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js:7]: Works for me too 👍.

**Review by AndrewPrigorshnev — APPROVED** [review]

File: plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js (hunk 1 above)
Comment (AndrewPrigorshnev) [reply plugins/chat/assets/javascripts/discourse/components/chat/admin/export-messages.js:7]: Works for me too 👍.

**Pull request merged**